src/mixing/mixing.py

```diff
@@ -99,15 +119,16 @@ def main_loop(self):
                 time.sleep(0)
```

Comment:
While we're here, move this `time.sleep(0)` outside of this `try`/`except` so that it definitely gets executed once per loop. Otherwise when there's nothing in the queues to move, this thread can starve the other ones.